packages/@react-spectrum/tag/docs/TagGroup.mdx

```diff
@@ -0,0 +1,224 @@
+      <Item key={item.key}>
+        {item.name}
+      </Item>
+    )}
+  </TagGroup>
+</Flex>
+```
+
+You can also disable specific tags by providing an array of keys to TagGroup via the `disabledKeys` prop. This will disable all interactions on disabled tags.
+
+```tsx example
+function AppList() {
+const defaultItems = [
+  {id: 1, name: 'Adobe Photoshop'},
+  {id: 2, name: 'Adobe XD'},
+  {id: 3, name: 'Adobe InDesign'},
+  {id: 4, name: 'Adobe AfterEffects'},
+  {id: 5, name: 'Adobe Illustrator'},
+  {id: 6, name: 'Adobe Lightroom'},
+  {id: 7, name: 'Adobe Premiere Pro'},
+  {id: 8, name: 'Adobe Fresco'},
+  {id: 9, name: 'Adobe Dreamweaver'}
+];
+
+  const [items, setItems] = React.useState(defaultItems);
+  const removeItem = (tag) => {
+    const updatedItems = items.filter(item => item.name !== tag);
+    setItems(updatedItems);
+  }
+
+  return (
+    <Flex width={500}>
+      <TagGroup items={items} disabledKeys={[2,4,5,8]} isRemovable onRemove={removeItem} aria-label="Disable individual tags example">
+        {(item) => (
+          <Item key={item.key}>
+            {item.name}
+          </Item>
+        )}
+      </TagGroup>
+    </Flex>
+  );
+}
+
+```
+
+### Removing Tags
+Adding the `isRemovable` prop will allow the tags to be removed from the group. The `onRemove` callback is then
+used to handle the remove logic.
+
+```tsx example
+function AppList() {
+const defaultItems = [
+  {id: 1, name: 'Adobe Photoshop'},
+  {id: 2, name: 'Adobe XD'},
+  {id: 3, name: 'Adobe InDesign'},
+  {id: 4, name: 'Adobe AfterEffects'},
+  {id: 5, name: 'Adobe Illustrator'},
+  {id: 6, name: 'Adobe Lightroom'},
+  {id: 7, name: 'Adobe Premiere Pro'},
+  {id: 8, name: 'Adobe Fresco'},
+  {id: 9, name: 'Adobe Dreamweaver'}
+];
+
+  const [items, setItems] = React.useState(defaultItems);
+  const removeItem = (tag) => {
+    const updatedItems = items.filter(item => item.name !== tag);
+    setItems(updatedItems);
+  }
+
+  return (
+    <Flex width={500}>
+      <TagGroup items={items} isRemovable onRemove={removeItem} aria-label="Removable TagGroup example">
+        {(item) => (
+          <Item key={item.key}>
+            {item.name}
+          </Item>
+        )}
+      </TagGroup>
+    </Flex>
+  );
+}
+
+```
+
+### Overflow
+If the length of the text is longer than the width of the component, the text will truncate with an ellipsis.
+
+```tsx example
+const items = [
+  {id: 1, name: 'Adobe Photoshop'},
+  {id: 2, name: 'Adobe XD'},
+  {id: 3, name: 'Adobe InDesign'},
+  {id: 4, name: 'Adobe AfterEffects'},
+  {id: 5, name: 'Adobe Illustrator'},
+  {id: 6, name: 'Adobe Lightroom'},
+  {id: 7, name: 'Adobe Premiere Pro'},
+  {id: 8, name: 'Adobe Fresco'},
+  {id: 9, name: 'Adobe Dreamweaver'}
+];
+
+<Flex width={100}>
+  <TagGroup items={items} aria-label="Overflow TagGroup example">
+    {(item) => (
+      <Item key={item.key}>
+        {item.name}
+      </Item>
+    )}
+  </TagGroup>
+</Flex>
+```
+
+## Props
+
+<PropTable component={{
```

Comment:
I'm seeing `onSelectionChange` in the table, but I don't think that should be there. Also it looks like `disabledKeys`, `isDisabled`, `isRemovable`, and `onRemove` all needs descriptions.